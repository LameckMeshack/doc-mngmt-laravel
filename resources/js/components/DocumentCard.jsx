import React from "react";
import { FcFile } from "react-icons/fc";
function DocumentCard({ name, access, department }) {
    return (
        <div className="document-card">
            <div className="document-card-left">
                <FcFile className="document-card-left-icon" />
            </div>
            <div className="document-card-right">
                <h5>{name}</h5>
                <small>{access}</small>
                <small> {department}</small>
            </div>
        </div>
    );
}

export default DocumentCard;
