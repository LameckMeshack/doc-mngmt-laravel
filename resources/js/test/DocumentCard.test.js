import React from "react";
import TestRenderer from "react-test-renderer";
import DocumentCard from "../components/DocumentCard";

test("DocumentCard", () => {
    const tree = TestRenderer.create(<DocumentCard />).toJSON();
    expect(tree).toMatchSnapshot();
});
