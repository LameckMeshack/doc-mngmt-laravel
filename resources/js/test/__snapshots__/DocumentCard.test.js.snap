// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocumentCard 1`] = `
<div
  className="document-card"
>
  <div
    className="document-card-left"
  >
    <svg
      className="document-card-left-icon"
      enableBackground="new 0 0 48 48"
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        {
          "color": undefined,
        }
      }
      version="1"
      viewBox="0 0 48 48"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <polygon
        fill="#90CAF9"
        points="40,45 8,45 8,3 30,3 40,13"
      />
      <polygon
        fill="#E1F5FE"
        points="38.5,14 29,14 29,4.5"
      />
    </svg>
  </div>
  <div
    className="document-card-right"
  >
    <h5 />
    <small />
    <small>
       
    </small>
  </div>
</div>
`;
